### ISSUE TITLE
'NoneType' object has no attribute 'get_repo'

### ISSUE THREAD

**Issue by travula** [vlead/translators#12]
@sounak98 @arghyatiger @talsperre 
I deployed translators from develop branch after the merge.  When I do a `post`, I get the following error
```
'NoneType' object has no attribute 'get_repo'
```

Is this due to a proxy setting?

**Comment by sounak98** [comment]
Yes, this is probably due to a proxy issue. Can you recheck and let me know?
